### PR TITLE
Avoid some uses of len-1 tuples.

### PR DESCRIPTION
**Pull request by anntzer**
... when scalars or f-strings or other constructs would do.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/offsetbox.py [hunk 1]
```diff
@@ -91,15 +91,14 @@ def _get_packed_offsets(wd_list, total, sep, mode="fixed"):
...
-    # d_list is currently not used.
+    w_list, d_list = zip(*wd_list)  # d_list is currently not used.
+    cbook._check_in_list(["fixed", "expand", "equal"], mode=mode)
 
     if mode == "fixed":
         offsets_ = np.cumsum([0] + [w + sep for w in w_list])
         offsets = offsets_[:-1]
         if total is None:
             total = offsets_[-1] - sep
-        return total, offsets
```
Comment (timhoffm) [inline lib/matplotlib/offsetbox.py]: I have a slight preference for keeping these. It's simpler to follow the code when returning early.

Comment (anntzer) [reply lib/matplotlib/offsetbox.py]: I wouldn't have made the change except for the fact that the function immediately below (_get_aligned_offsets) uses single-return-at-end-of-if-elif-block, so I made both of them consistent.

Comment (timhoffm) [reply lib/matplotlib/offsetbox.py]: The difference is that the function below has typically only two lines per case which are just assignments (not much going on there), but here we have 4-9 lines with conditional logic per case. IMHO that justifies different approaches.

Comment (anntzer) [reply lib/matplotlib/offsetbox.py]: ok, restored

**Pull request merged**